tests/test_check.py

```diff
@@ -193,7 +223,7 @@ def test_check_with_config():
 
 
 def test_check_saves_file() -> None:
-    """Test whether or not specifiying an output properly saves a file"""
+    """Test whether or not specifying an output properly saves a file"""
```

Comment:
See my above comment about not making changes unrelated to your assigned task in this PR. Same with the change below. These changes should be both be reverted.